.github/workflows/triage-label-pr.yml

```
@@ -1,5 +1,5 @@
 # https://github.com/marketplace/actions/labeler
-name: Add labels
+name: triage-labels-pr
```

```suggestion
name: triage-pr-labels
```
?

To be consistent with other.